stixcore/products/product.py

```diff
@@ -169,7 +169,7 @@ def __call__(self, *args, **kwargs):
                     try:
                         energies = QTable.read(file_path, hdu='ENERGIES')
                     except KeyError:
```

Comment:
should here also the new read table wrapper function be used?

Reply:
Possibly I don't think we ever use the energies which are read we always recreate when writing but no harm to use it.

Reply:
✅